src/components/SearchTracePage/index.js

```diff
@@ -218,20 +220,32 @@ SearchTracePage.propTypes = {
   errorMessage: PropTypes.string,
 };
 
+const stateTraceXformer = getLastXformCacher(stateTrace => {
+  const { traces: traceMap, loading, error: traceError } = stateTrace.toJS();
```

Comment:
Q (curious): how is `stateTrace` represented in the first place? What does the call to `toJS()` achieve?